src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery balls = new Lottery();
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            System.out.println(new Lottery().getRandomBall());
+        }
```

Comment:
Try not to create variables that you don't use.
```suggestion
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(new Lottery().getRandomBall());
        }
```
Or if you have already created it, you can use the following
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```